pr-data.csv

```diff
@@ -232,10 +232,10 @@ https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-t
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner-blink,org.apache.flink.table.planner.expressions.ScalarFunctionsTest.testStringToMap,ID,Rejected,https://github.com/apache/flink/pull/11286,
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner-blink,org.apache.flink.table.planner.expressions.SqlExpressionTest.testArithmeticFunctions,ID,,,
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner,org.apache.flink.table.runtime.harness.AggFunctionHarnessTest.testCollectAggregate,ID,,,
-https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-runtime-blink,org.apache.flink.table.runtime.operators.bundle.MapBundleOperatorTest.testSimple,ID,Opened,https://github.com/apache/flink/pull/11268,
-https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-runtime-blink,org.apache.flink.table.runtime.operators.join.ProcTimeBoundedStreamJoinTest.testProcTimeInnerJoinWithCommonBounds,ID,Opened,https://github.com/apache/flink/pull/11287,
+https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-runtime-blink,org.apache.flink.table.runtime.operators.bundle.MapBundleOperatorTest.testSimple,ID,Accepted,https://github.com/apache/flink/pull/11268,
+https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-runtime-blink,org.apache.flink.table.runtime.operators.join.ProcTimeBoundedStreamJoinTest.testProcTimeInnerJoinWithCommonBounds,ID,Accepted,https://github.com/apache/flink/pull/11287,
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-runtime-blink,org.apache.flink.table.runtime.operators.join.RowTimeBoundedStreamJoinTest.testRowTimeFullOuterJoin,ID,,,https://github.com/TestingResearchIllinois/idoft/issues/20
-https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-runtime-blink,org.apache.flink.table.runtime.operators.join.RowTimeBoundedStreamJoinTest.testRowTimeInnerJoinWithCommonBounds,ID,Opened,https://github.com/apache/flink/pull/11287,
+https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-runtime-blink,org.apache.flink.table.runtime.operators.join.RowTimeBoundedStreamJoinTest.testRowTimeInnerJoinWithCommonBounds,ID,Accepted,https://github.com/apache/flink/pull/11287,
```

Comment:
https://github.com/apache/flink/pull/11268 and https://github.com/apache/flink/pull/11287 are closed and not accepted